doc/api/http.md

```diff
@@ -632,7 +632,7 @@ See also: [`request.setTimeout()`][].
 added: v0.1.94
 -->
 
-* `response` {http.IncomingMessage}
+* `request` {http.IncomingMessage}
```

Comment:
This is not correct. The relevant code is [here](https://github.com/nodejs/node/blob/be93b7a58221f3886efdb4f022dfe3649c97e93b/lib/_http_client.js#L576).